freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -215,6 +215,15 @@ def _get_bundles_to_rebuild(self):
             # images in related images
             bundle_nvrs = self._get_impacted_bundles(original_digests_by_nvr.values())
 
+        def is_hotfix(nvr):
```

Comment:
It's unnecessary to introduce a nested function, but I'm fine with this.

Reply:
Ahh, I used a separate function because I didn't know if you could implement a logger within a list comprehension. I don't mind changing it but I don't see anything online about that

Reply:
You don't have to insist with list comprehension, you can do it in loop:
```
for nvr in bundle_nvrs[:]:
    if self._pyxis.is_hotfix_image(nvr):
        bundle_nvrs.remove(nvr)
        log.debug("Excluding hotfix bundle: %s", nvr)
```

Reply:
I'm not sure why I was so insistent on using a list comprehension :sweat_smile: , but I changed it to a normal loop as suggested (it's definitely cleaner than using nested funcs)